src/unity/toolkits/activity_classification/sequence_iterator.cpp

```diff
@@ -144,7 +145,7 @@ variant_map_type _activity_classifier_prepare_data_impl(const gl_sframe &data,
 
     // Build a dict pf the column order by column name, to later access within the iterator
     auto column_index_map = generate_column_index_map(data.column_names());
-
+    
```

Comment:
Nit: drop this whitespace change. It looks like we prefer no trailing whitespace here....